data/json/species.json

```diff
@@ -72,6 +72,13 @@
     "anger_triggers": [ "FRIEND_DIED" ],
     "fear_triggers": [ "HURT", "FIRE" ]
   },
+  {
+    "type": "SPECIES",
+    "id": "CENTIPEDE",
+    "description": "a centipede",
+    "footsteps": "skittering.",
```

Comment:
If mega/armored ones also had PLAYER_CLOSE for anger, that could be put here instead.